include/lmms_math.h

```diff
@@ -342,23 +205,19 @@ constexpr T lerp(T a, T b, F t)
 // @note Once we upgrade to C++20, we could probably use std::formatted_size
 static inline int numDigitsAsInt(float f)
 {
-	// use rounding:
-	// LcdSpinBox sometimes uses roundf(), sometimes cast rounding
-	// we use rounding to be on the "safe side"
```

Comment:
I realised I approved this earlier, but I think this comment might be worth keeping.

Reply:
added back in